agents/diamagent.go

```diff
@@ -84,7 +86,18 @@ func (da *DiameterAgent) handlers() diam.Handler {
 		ProductName:      datatype.UTF8String(da.cgrCfg.DiameterAgentCfg().ProductName),
 		FirmwareRevision: datatype.Unsigned32(utils.DIAMETER_FIRMWARE_REVISION),
 	}
+	interfaces, _ := net.Interfaces()
```

Comment:
Please treat the error here and also add support for configurable listen addresses